### PR TITLE
Lesson5: Grammar and text tweaks

### PR DESCRIPTION
**Pull request by briangershon**
Ran through Lesson 5 and made some initial fixes.

### CODE REVIEW

File: apps/academy/src/pages/tracks/nft-solidity/5.mdx
Comment (briangershon) [inline apps/academy/src/pages/tracks/nft-solidity/5.mdx:17]: I assume this should be `ropats16` to also match the Twitter?

Comment (elPiablo) [reply apps/academy/src/pages/tracks/nft-solidity/5.mdx:17]: Yes, for sure. thanks :) We added new components, and they got populated with 'something' or 'someone'. They all needed to be reviewed.

File: apps/academy/src/pages/tracks/nft-solidity/5.mdx
Comment (elPiablo) [inline apps/academy/src/pages/tracks/nft-solidity/5.mdx:151]: Hehe, we had a conversation about this. The final decision was to use front-end hyphenated in texts, headers and descriptions, and `frontend` in building the dApp in the lesson, or building Academy's architecture.

**Review by elPiablo — APPROVED** [review]
Brilliant stuff, @briangershon. You can merge this for sure, if you like. It will help me with the quizzes tasks 😀 I'll tick 'Approve'. If you just want to re-assign me instead, go for it. I trust you, and learn from your workflow, whatever that is. Cheers!

File: apps/academy/src/pages/tracks/nft-solidity/5.mdx
Comment (briangershon) [reply apps/academy/src/pages/tracks/nft-solidity/5.mdx]: I switched this back to `Front-end`

**Review by elPiablo — COMMENTED** [review]
wey hey!! good to go 😄

**Pull request merged**
